cmake/iotjs.cmake

```diff
@@ -580,6 +589,7 @@ if(NOT BUILD_LIB_ONLY)
           RUNTIME DESTINATION "${INSTALL_PREFIX}/bin"
           LIBRARY DESTINATION "${INSTALL_PREFIX}/lib"
           PUBLIC_HEADER DESTINATION "${INSTALL_PREFIX}/include/iotjs")
+  install(TARGETS ${TARGET_LIB_IOTJS} DESTINATION ${LIB_INSTALL_DIR}) 
```

Comment:
The install command can take multiple target arguments, I'd suggest to not duplicate the calls. The install call on line 588 could be modified to be `install(TARGETS ${TARGET_IOTJS} ${TARGET_LIB_IOTJS} ...`, and then the calls on line 592 and 594 could be removed.

Reply:
yes you're right, I am updating my patch and will update the other one that use destdir